games-strategy/opendune/opendune-0.9.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Open-source clone of Dune II: The Battle for Arrakis"
+DESCRIPTION="An open source re-creation of the popular game \"Dune II\", \
+originally made by Westwood Studios, and released by Virgin Entertainment.
+It attempts to re-create the original game and apply modern technology to \
+it to allow it to be run natively on most operating systems.
+
+Please note that you must provide the original Dune II 1.07 data files in \
+/boot/home/config/non-packaged/data/opendune directory \
+for the game to run properly."
+HOMEPAGE="http://www.opendune.org/"
+COPYRIGHT="2009-2018 OpenDUNE Developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/OpenDUNE/OpenDUNE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="8d5ed67669df1f17a44c097d9b6bee4e0623ce2a37f11938ce9cd77de546d06c"
+SOURCE_FILENAME="OpenDUNE-$portVersion.tar.gz"
+SOURCE_DIR="OpenDUNE-$portVersion"
+PATCHES="opendune-$portVersion.patchset"
+ADDITIONAL_FILES="opendune.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/opendune.ini keep-old
+	"
+USER_SETTINGS_FILES="
+	settings/opendune directory
+	settings/opendune/savegames directory
+	settings/opendune/opendune.ini template settings/opendune.ini
+	"
+
+PROVIDES="
+	opendune$secondaryArchSuffix = $portVersion
+	app:OpenDUNE = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+	lib:libSDL_image$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL$secondaryArchSuffix
+	devel:libSDL_image$secondaryArchSuffix
```

Comment:
While we are at it, does it build with SDL1.2 then maybe check requirements here: https://github.com/haikuports/haikuports/issues/3878